go.mod

```diff
@@ -1,10 +1,12 @@
 module github.com/go-rel/sqlite3
 
-go 1.20
+go 1.21
```

Comment:
With installed go 1.21 this was the only way I could get to go mod tidy to work :/

Reply:
oh, that's because go-rel has go 1.21 as min version set